redisson/src/main/java/org/redisson/spring/cache/RedissonCache.java

```diff
@@ -46,11 +46,14 @@ public class RedissonCache implements Cache {
     private final AtomicLong puts = new AtomicLong();
     
     private final AtomicLong misses = new AtomicLong();
+
+    private AtomicLong evictions;
```

Comment:
please make it final and init in field.

Reply:
I was initialising it in one of the constructor and not in the other one as I wanted to signal that in one case we can count evictions but not in the other one. Does it makes sense?

Reply:
Sorry, which case? I think we need to calculate it the same way we do for other metrics

Reply:
Oh yes I see, I was wrong with my assumption, sorry, I'll fix this